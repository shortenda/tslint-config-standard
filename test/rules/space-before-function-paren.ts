function foo() {
  return
}

let bar = function() {
  return
}

class Foo {
  constructor() {
    return
  }

  namedFunction() {
    return
  }
}

// space is option between async and parens
let foo = async () => 1
let foo2 = async() => 1
