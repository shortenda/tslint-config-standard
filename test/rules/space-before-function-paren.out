
space-before-function-paren.ts[1, 13]: Missing whitespace before function parens
space-before-function-paren.ts[10, 14]: Missing whitespace before function parens
space-before-function-paren.ts[14, 16]: Missing whitespace before function parens
